Resources/public/javascript/countable.js

```diff
@@ -0,0 +1,25 @@
+$(function(){
+  $('.js-countable').each(function(index, item){
+      item = $(item);
+      let count_area =  item.find('textarea');
+      let counter_attr = count_area.attr('data-length');
+      let total = counter_attr - count_area.val().length;
+
+      let count_text =  item.find('.js-countable-total');
+      count_text.text(total);
+      if(isNaN(total) || total <= 0) {
+        count_text.addClass('c-counter-error');
+      }else {
+        count_text.removeClass('c-counter-error');
+      }
+      count_area.on('keyup', function(){
+            let total = counter_attr - count_area.val().length;
+            count_text.text(total);
+            if(isNaN(total) || total <= 0) {
+              count_text.addClass('c-counter-error');
+            }else {
+              count_text.removeClass('c-counter-error');
+            }
+      })
+  });
+});
```

Comment:
**Code duplication**: your doing the same things twice. Abstract the 'keyup' function so you can call it as an event listener but also call it right after init.
And instead of using an if(<CONDITION) with addClass/removeClass, you can use toggleClass('classname', <CONDITION>).

Also notice that it only applies to textarears, shouldnt it also work for inputs?